### DISCUSSION TITLE
What is Trickplay?

### DISCUSSION THREAD

**Discussion by MoonOverMiami** [jellyfin/jellyfin#12632] [General]
What is Trickplay?

**Comment by sjorge** [comment]
It generates a series of tiny preview images for all your media. 

These can be used by clients that support trickplay to show a preview while skipping forwards/backworss through the video file.

**Reply by junadlao** [reply]
tnx for this info. But shouldn't this just default to the already existing chapter images?

**Reply by gnattu** [reply]
It captures more frames than chapter images so it need to be regenerated

**Reply by Jeffofmany** [reply]
Finally, a simple explination somewhere in all the junk on the internet.

**Reply by swever826** [reply]
Is there a reason this warning only appears on the Chapter Images section?
`The process can be slow, resource intensive, and may require several gigabytes of space.`

Is this not the case for Trickplay?
